### ISSUE TITLE
Unknown error: Failed to fetch on docker instance

### ISSUE THREAD

**Issue by Terramoto** [zadam/trilium-web-clipper#45]
Hello,

I've just recently setup a Trilium container on a random port 32768. I'm trying to set it up on the extension but it's coming up with:
Unknown error: Failed to fetch on docker instance
![image](https://user-images.githubusercontent.com/5443354/192156322-567ff5ab-aa28-48d4-8e48-66e0e9930c05.png)

With or without port being specified it generates the same error. It's like it's not even reaching the server.

**Comment by zadam** [comment]
> It's like it's not even reaching the server.

Looks like that, yes. Does Trilium load when you load 10.0.0.51:32768 in the browser?

**Comment by Terramoto** [comment]
> > It's like it's not even reaching the server.
> 
> Looks like that, yes. Does Trilium load when you load 10.0.0.51:32768 in the browser?

Yep, i can access it and create notes. Not sure if it's because im running it on Brave Browser which is chromium based.
It looks like the error is triggered by these lines in options.js:
`    try {
        resp = await fetch($triliumServerUrl.val() + '/api/login/token', {
            method: "POST",
            headers: {
                'Accept': 'application/json',
                'Content-Type': 'application/json'
            },
            body: JSON.stringify({
                username: $triliumServerUsername.val(),
                password: $triliumServerPassword.val()
            })
        });
    }`

After including http:// in the server field it accepted the server, so everything working now.
